.gitignore

```diff
@@ -36,4 +36,4 @@ Session.vim
 
 .history
 
-values.yaml
+/values.yaml
```

Comment:
This commit can be dropped now after rebasing on latest master, it was only temporarily.

Reply:
rebased